kohese-portal/client/src/components/type-editor/view-model-editor/view-model-editor.component.ts

```diff
@@ -196,9 +196,12 @@ export class ViewModelEditorComponent {
     return Object;
   }
 
-  public constructor(private _changeDetectorRef: ChangeDetectorRef,
-    private _dialogService: DialogService, private _itemRepository:
-    ItemRepository) {
+  public constructor(
+                      private _changeDetectorRef: ChangeDetectorRef,
+                      private _dialogService: DialogService,
+                      private _itemRepository: ItemRepository,
+  ) {
+    UuidV1();
```

Comment:
What does this line do?  Seems to be calling the function and throwing away the result.

Reply:
This sets the default instead of saying UUidV1().default(). If you try to set the default the old way like that then you get an error saying something like `ERROR: default().default() is not a function`